app/assets/stylesheets/lux/_variables.scss

```diff
@@ -125,7 +125,7 @@ $facet-line-height:           1.5rem;
 
 $facet-label-font-size:       $font-size-base * 0.9375;
 $facet-label-line-height:     $facet-label-font-size * 1.2;
-$facet-top-margin:            2rem;
+$facet-top-margin:            $search-form-margin-x;
```

Comment:
Can we make a different variable for this? If we decide to change the x-margins on the search form, it would have unintended consequences for this unrelated margin.

Reply:
Will do.